### PR TITLE
added IDC collections statistics query, named  query_stats_agg_idc.txt in the root folder.

### PR DESCRIPTION
**Pull request by ccosmin97**
@fedorov

### CODE REVIEW

File: query_stats_agg_idc.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  SeriesInstanceUID,
+  StudyInstanceUID,
+  PatientID,
+  RepetitionTime,
+  EchoTime,
+  SliceThickness,
+  MagneticFieldStrength,
+  PixelSpacing[SAFE_OFFSET(0)] as PixelSpacing,
+  CASE
+    WHEN SeriesDescription LIKE '%t2_tse_tra%' AND collection_id = "prostatex" AND Modality = "MR" THEN "t2w_prostateX"
```
Comment (fedorov) [inline query_stats_agg_idc.txt]: I would suggest to have a column that just defines the contrast of the series (e.g., "T2W"), and not mix contrast type with the collection ID. We already have collection ID in an existing column.

Comment (ccosmin97) [reply query_stats_agg_idc.txt]: ok, i will separate collection_id and SeriesDescription, then group by colllection_id, SeriesDescription.

**Comment by fedorov** [comment]
Just to motivate you to improve your git skills:
1. Rebase the commits into a single commit
2. While rebasing, update the commit message to follow https://slicer.readthedocs.io/en/latest/developer_guide/contributing.html
3. Force-push the local content to your branch

@vkt1414 can answer questions! ;-)

**Comment by ccosmin97** [comment]
> Just to motivate you to improve your git skills:
> 
> 1. Rebase the commits into a single commit
> 2. While rebasing, update the commit message to follow https://slicer.readthedocs.io/en/latest/developer_guide/contributing.html
> 3. Force-push the local content to your branch
> 
> @vkt1414 can answer questions! ;-)

git rebase -i HEAD~3

File: query_stats_agg_idc.txt [hunk 2]
```diff
@@ -0,0 +1,48 @@
+WITH temp_query as (
```
Comment (fedorov) [inline query_stats_agg_idc.txt]: You can add a comment header explaining what this query does, and even explain individual lines that are complex. Think about yourself when you started working with SQL - would this query be straightforward to understand?

**Review by fedorov — CHANGES_REQUESTED** [review]
See earlier comments.

**Comment by fedorov** [comment]
> Rebase the commits into a single commit

@ccosmin97 please read this: https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line

**Pull request merged**